### ISSUE TITLE
NVDA reads Details List non-actionable headers as Clickable.

### ISSUE THREAD

**Issue by mdmdakbar** [OfficeDev/office-ui-fabric-react#8496]
NVDA reading details list non-actionable header as Clickable in scan mode.

Browsers affected: Chrome, Firefox

Also Narrator has somewhat similar issue with Edge: https://github.com/OfficeDev/office-ui-fabric-react/issues/4848

Impact:
User with visual impairments will not be able to know that the headers are non interactive elements

Actual behavior:
NVDA should not announce the header as clickable as the header is not actionable.

Repro:
https://developer.microsoft.com/en-us/fabric#/components/detailslist

Please follow the details list example in which the header is non-actionable to repro the issue

**Comment by dzearing** [comment]
@mdmdakbar Why is this the case? Many screen reader issues are due to the screen reader reading things poorly, rather than the component rendering the wrong attributes (or missing them.)

Is there a particular DOM change we should be changing that is causing this? Or is this a screen reader issue?

@cschlechty @ThomasMichon @KevinTCoughlin fyi

**Comment by KevinTCoughlin** [comment]
Some of the header elements are actionable, particularly the "Select all" first column if selection is enabled. The other header elements are actionable for sorting if enabled. This is probably why they are currently interactive.

**Comment by Vitalius1** [comment]
@mdmdakbar, the link you provided where to repro the behavior you stated above is actually behaving as it should be because of all the headers are actually having the sorting enabled on them.
